spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/reload/ConfigurationUpdateStrategy.java

```diff
@@ -47,10 +45,7 @@ public void reload() {
 
 	@Override
 	public String toString() {
-		final StringBuilder sb = new StringBuilder("ConfigurationUpdateStrategy{");
-		sb.append("name='").append(this.name).append('\'');
-		sb.append('}');
-		return sb.toString();
+		return "ConfigurationUpdateStrategy{name='" + this.name + "'}";
```

Comment:
far simpler `toString` and `StringBuilder` is going to be generated by the compiler _anyway_; even in java-9 where `String::concat` compiles as `invokedynamic`, but will still _use_ `StringBuilder::append`, by default.